src/UglyToad.PdfPig/Graphics/ReflectionGraphicsStateOperationFactory.cs

```diff
@@ -129,6 +129,10 @@ public IGraphicsStateOperation Create(OperatorToken op, IReadOnlyList<IToken> op
                 case SetColorRenderingIntent.Symbol:
                     return new SetColorRenderingIntent((NameToken)operands[0]);
                 case SetFlatnessTolerance.Symbol:
+                    if (operands.Count == 0)
```

Comment:
Wonder if there's a way to change this mapping to use new codegen stuff where the input is basically expected args, fallback behavior, expected output or something, to reduce the number of times we need to fix individual ones to be more fault tolerant, ah well, a nice thought experiment, LGTM